src/algebra/ordered_monoid_lemmas.lean

```diff
@@ -325,7 +329,7 @@ calc  a * b < 1 * b : mul_lt_mul_right' ha b
         ... < c     : hb
 
 /-- Assumes right covariance. -/
-@[to_additive left.add_nonneg]
+@[to_additive right.add_nonneg]
```

Comment:
While you're here, shouldn't the lemma below be `add_pos` and not `pos_add`?

Reply:
Thanks, Eric: I changed the name below!